Eastern-Research-Group/csb-rebate-forms-app
# Feature/re organize utilities

## Related Issues:
* CSBAPP-91

## Main Changes:
* Moves most custom hooks and functions out of config file and components into utilities file
* Updates "New Payment Request" button functionality to prevent duplicate submits

## Steps To Test:
1. Navigate to dashboard and click the "New Payment Request" button several times (with the browser's network tab open to see network requests). Only one POST request should occur, limiting the creation of a single new payment request, no matter how many times you click the button. As soon as the request succeeds and data is returned from the server, you're still re-directed to the payment request submission.
